models/zendesk__ticket_backlog.sql

```diff
@@ -0,0 +1,99 @@
+with ticket_field_history as (
```

Comment:
enable this model only if status is in field_history list

Reply:
Updated.